examples2/src/erc20.rs

```diff
@@ -629,25 +777,159 @@ mod tests {
         pobcoin.burn_and_get_paid(100.into());
         assert_eq!(env.balance_of(&alice), current_balance + U512::from(100));
 
+        env.print_gas_report()
+    }
+
+    #[test]
+    fn erc20_call_result() {
+        let env = odra2::test_env();
+        let alice = env.get_account(0);
+        let bob = env.get_account(1);
+
+        // Deploy the contract as Alice.
+        let erc20 = Erc20Deployer::init(&env, Some(100.into()));
+        let pobcoin = Erc20Deployer::init(&env, Some(100.into()));
+
+        // Make a call or two
+        erc20.transfer(bob, 10.into());
+        erc20.transfer(bob, 30.into());
+
+        // Test call result
+        assert_eq!(env.get_events_count(&erc20.address), 2);
+
+        let call_result = env.last_call_result();
+        assert!(call_result.result.is_ok());
+        assert_eq!(call_result.contract_address, erc20.address);
+        assert_eq!(call_result.caller, alice);
+        assert_eq!(
+            call_result.result.unwrap(),
+            Bytes::from(().to_bytes().unwrap())
```

Comment:
Would be easier to convert an empty vec into Bytes.
```rust
vec![].into()
```

Reply:
Done